Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -75,45 +76,24 @@ + (BNCPreferenceHelper *)getInstance {
 
 #pragma mark - Debug methods
 
-+ (void)setDebug {
-    [BNCPreferenceHelper getInstance].isDebugMode = YES;
++ (void)setDebug:(BOOL)debug {
+    [BNCPreferenceHelper getInstance].isDebugMode = debug;
 }
 
 + (void)clearDebug {
     [BNCPreferenceHelper getInstance].isDebugMode = NO;
-    
-    [self disconnectRemoteDebug];
-}
-
-+ (void)connectRemoteDebug {
-    serverInterface = [[BranchServerInterface alloc] init];
-    bnc_asyncLogQueue = dispatch_queue_create("bnc_log_queue", NULL);
-
-    [serverInterface connectToDebugWithKey:Branch_Key callback:^(BNCServerResponse *response, NSError *error) {
-        if (error) {
-            NSLog(@"Failed to connect to debug: %@", error);
-        }
-        else {
-            [BNCPreferenceHelper getInstance].isConnectedToRemoteDebug = YES;
-            [bnc_asyncDebugConnectionDelegate debugConnectionEstablished];
-        }
-    }];
-}
-
-+ (void)disconnectRemoteDebug {
-    BNCPreferenceHelper *preferenceHelper = [BNCPreferenceHelper getInstance];
-    if (preferenceHelper.isConnectedToRemoteDebug) {
-        preferenceHelper.isConnectedToRemoteDebug = NO;
-        
-        [serverInterface disconnectFromDebugWithKey:Branch_Key callback:NULL];
-    }
+    [BNCPreferenceHelper getInstance].isConnectedToRemoteDebug = NO;
```

Comment:
`clearDebug` is basically shorthand for calling `setDebug:NO` `setRemoteDebug:NO`. Could be removed, just something that already existed that I didn't change.